packages/slice-machine/tests/project/customtypes/about/index.json

```diff
@@ -14,6 +14,15 @@
           "single": "paragraph,preformatted,heading1,heading2,heading3,heading4,heading5,heading6,strong,em,hyperlink,image,embed,list-item,o-list-item,rtl"
         }
       },
+      "Am-i-cool-or-not": {
```

Comment:
maybe we should avoid commit our local test on slice / custom types no ? To avoid adding confusion in PR